devel/apache-ant/Portfile

```diff
@@ -52,6 +53,11 @@ variant source description "build from source; not recommended" {
         build.cmd                       ./build.sh
         build.args                      -Dchmod.fail=false -Ddist.name=${name}
         build.target                    dist
+
+        post-patch {
+            # Don't fail on Javadoc errors
+            reinplace "s|failonerror=\"true\"||g" ${worksrcpath}/build.xml
```

Comment:
It would generally be better to have this be a patch so it doesn't match on new instances of this string.